src/sentry/integrations/gitlab/webhooks.py

```diff
@@ -112,7 +111,7 @@ def __call__(self, integration, organization, event):
                 extra={"integration_id": integration.id, "error": str(e)},
             )
 
-        if not author_email:
+        if not author_email or len(author_email) > 75:
```

Comment:
@mgaeta Sending a 404 here when the email is too long is very deceiving. Why don't we just fix the column so it can be longer than 75 characters?